pl_examples/loop_examples/yielding_training_step.py

```diff
@@ -88,8 +88,9 @@ def _training_step(self, generator):
         training_step_output = next(generator)
         self.trainer.training_type_plugin.post_training_step()
 
-        training_step_output = self.trainer.call_hook("training_step_end", training_step_output)
-
+        model_output = self.trainer._call_lightning_module_hook("training_step_end", training_step_output)
+        ttp_output = self.trainer._call_ttp_hook("training_step_end", training_step_output)
+        training_step_output = ttp_output if model_output is None else model_output
```

Comment:
nit: keep the whitespace

```suggestion
        training_step_output = ttp_output if model_output is None else model_output

```